### PR TITLE
Max 15

### PR DESCRIPTION
**Pull request by MassimoVilla**
Dear all, here comes MS BDLaethd16, an 18th-cent. splendidly (to my unprofessional eye) illuminated manuscript of poetry.
Of course, I just gave some basic info on the binding: Eliana will provide a more thorough description of these features. I've completely left the decoDesc to Jacopo.

### CODE REVIEW

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, great!
I wonder whether it might be useful to very summarily describe the text/image layout explicitly in one sentence, maybe in layout or decorations? Right now, I'm not sure that someone who reads only the description would be able to reconstruct it.

**Review by ElianaDalSasso — CHANGES_REQUESTED** [review]
Thank you Massimo! It a really interesting manuscript to my eyes as well :)
I have just two minor comments.

File: OxfordBodleian/Juel-Jensen/BDLaethd16.xml [hunk 1]
```diff
@@ -159,39 +246,125 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                            </item>
+                            <item xml:id="e2">
+                                <desc>Two white stickers are glued on the inner side of the back board.
+                                    On the right one is written "489" in black pen, on the left one is printed the number "1067".
+                                </desc>
+                            </item>
+                                <item xml:id="e3">
+                                <desc>
+                                    The incipits of stanzas 8 and 9 of the <ref target="#i1">"Saqoqāwa nafs"</ref> are indicated with Ethiopic numbers on the left margin
+                                    of <locus target="#1va"/>.  Clues for the rubricator are found on the left margin of <locus from="2ra" to="4ra"/>.
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethd16.xml:336]: Yes, exactly!

File: OxfordBodleian/Juel-Jensen/BDLaethd16.xml [hunk 2]
```diff
@@ -111,6 +189,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         </item>
                                     </list>
                                 </collation>
+                                <condition key="good">
+                                    The leather cover is slightly damaged, especially at the corners and along its edges.
+                                    The sewing is broken in correspondence of the centerfold of the <ref target="#q2">second quire</ref>.
+                                    The parchment is crumpled in correspondence of the outer corners of some leaves, especially at the beginning of the manuscript.
+                                    Some leaves stains of unspecified nature (e.g. <locus target="#33v #34r #35v"/>).
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethd16.xml:196]: Thank you Eliana! I have added these observations in the condition section.

File: OxfordBodleian/Juel-Jensen/BDLaethd16.xml [hunk 3]
```diff
@@ -159,39 +246,125 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                               indicated as <foreign xml:lang="gez">ገብርከ፡</foreign>, ‘your servant’.
+                               The name of a secondary owner, <persName ref="PRS13955WalattaS"/>, is indicated on <locus target="#2ra"/> on a previously erased name.
+                               The manuscript comes from the area of <placeName ref="wd:Q218861">Gondar</placeName>. It was taken by <roleName type="title">ʾAṣe</roleName> 
+                               <persName ref="PRS9430Tewodros"/> and entered the <placeName ref="INS0101MadhaneAlam"/> collection in <placeName ref="LOC4547Maqdal"/> (cp. <ref target="#a1"/>).
+                               The manuscript was then brought to <placeName ref="wd:Q21">England</placeName> after the 1868 expedition by <persName ref="PRS7484Napier"/>. 
+                               The subsequent owners are unknown in detail. The book was sold to the <placeName ref="INS0525Quaritch"/> and
+                               eventually purchased by <persName ref="PRS5782JuelJen"/> at an auction at Sotheby's on 20 March 1983, according to the auction catalogue.
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethd16.xml:372]: Exactly, thank you! I've created the Person record and added a note within History

**Review by ElianaDalSasso — APPROVED** [review]

**Comment by gnizla** [comment]
> Thank you, great! I wonder whether it might be useful to very summarily describe the text/image layout explicitly in one sentence, maybe in layout or decorations? Right now, I'm not sure that someone who reads only the description would be able to reconstruct it.

Hello @thea-m I could include this in the opening description of the decorations

**Comment by MassimoVilla** [comment]
> > Thank you, great! I wonder whether it might be useful to very summarily describe the text/image layout explicitly in one sentence, maybe in layout or decorations? Right now, I'm not sure that someone who reads only the description would be able to reconstruct it.
> 
> Hello @thea-m I could include this in the opening description of the decorations

I have added a short sentence in the layout section, inside `<note corresp="#textarea2 #margin2">`. We can discuss if it is more appropriate here or in the decoDesc part.

**Review by thea-m — APPROVED** [review]

**Comment by MassimoVilla** [comment]
Do you think we can merge this branch (and Max14)? I am afraid that, over time, the likelihood of having conflicting branches will increase

**Review by gnizla — APPROVED** [review]

**Pull request merged**